integration-tests/README.MD

```diff
@@ -0,0 +1,40 @@
+## Overview
+
+Integration tests for status-go
+
+## Table of Contents
+
+- [Overview](#overview)
+- [How to Install](#how-to-install)
+- [How to Run](#how-to-run)
+  - [Running Tests](#running-tests)
+- [Implementation details](#implementation-details)
+
+## How to Install
+
+* Install [Docker](https://docs.docker.com/engine/install/) and [Docker Compose](https://docs.docker.com/compose/install/)
+- Checkout all listed contract repositories into one folder: 
+    - [Status Network Token V2](https://github.com/status-im/status-network-token-v2)
+    - [Communities Contracts](https://github.com/status-im/communities-contracts?tab=readme-ov-file#installation-and-development)
+* Add environment variable, path to contract repos e.g. `export PATH_TO_CONTRACT_REPOS=~/directory-with-every-contract-repo`
+* Install [Python 3.10.14](https://www.python.org/downloads/)
+* In `integration-tests` directory, run `pip install -r requirements.txt`
+* **Optional (for test development)**: Use Python virtual environment for better dependency management. You can follow the guide [here](https://akrabat.com/creating-virtual-environments-with-pyenv/):
+
+## How to Run
+
+- In `integration-tests/tests` run `docker compose up --build`, as result:
```

Comment:
Do you think that there would be any value in explicitly telling the reader to navigate to `integration-tests/tests`?

Something like:

```suggestion
- Navigate to `integration-tests/tests`
- Run `docker compose up --build`, as result:
```